packages/venia-concept/src/components/RadioGroup/radio.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { node, shape, string } from 'prop-types';
+import { Radio } from 'informed';
+
+import classify from 'src/classify';
+import defaultClasses from './radio.css';
+
+/* TODO: change lint config to use `label-has-associated-control` */
+/* eslint-disable jsx-a11y/label-has-for */
```

Comment:
The plugin rule [`jsx-a11y/label-has-for`](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-for.md) is deprecated, but it's currently still part of [our lint config](https://github.com/magento-research/magento-eslint/blob/master/index.js). It's too rigid:

- it requires that any component rendering a `<label>` accept a prop named `htmlFor`
- it requires both nesting (`<label>` wraps `<input>`) and `id` + `htmlFor`
- somehow it doesn't work even when these criteria are met

So I'm disabling it. We'll switch to [`label-has-associated-control`](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md) at some point.